pkg/odo/genericclioptions/context.go

```diff
@@ -118,9 +118,11 @@ func getValidConfig(command *cobra.Command) (*config.LocalConfigInfo, error) {
 		// Gather nessasary info
 		p := command.Parent()
 		r := command.Root()
+		fmt.Println("root is", r.Name(), "parent is", p.Name())
```

Comment:
I suspect that this is debugging output left accidentally

Reply:
yeah :see_no_evil: